awesomplete.css

```diff
@@ -1,4 +1,14 @@
 [hidden] { display: none; }
+.visually-hidden {
+	border: 0;
+	clip: rect(0 0 0 0);
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	width: 1px;
+}
```

Comment:
Sorry, I hadn’t realized I was commenting on the wrong PR. 
This element doesn’t get focus, so a lot of the concerns described in the article don’t make sense here.
I don’t want to add current verbosity for potential future changes. 
`padding: 0` and `border:0` aren’t needed in any way. `margin: -1px` also shouldn’t be needed. The clip syntax needs to use commas, this is the IE < 7 clip syntax, and we don’t care about IE < 7.